buildenv/jenkins/README.md

```diff
@@ -100,152 +97,39 @@ Having a complicated regex in the pull request trigger is what allows us to laun
-        - Archives the SDK and test material for use in downstream jobs
-    - Trigger:
-        - This job is used in other pipelines but can be launched manually
-
-- Build-JDK10-aix_ppc-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Build-JDK10-linux_390-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Build-JDK10-aix_ppc-64_cmprssptrs)
-    - Description:
-        - Compiles java10 on aix_ppc-64_cmprssptrs
-        - Archives the SDK and test material for use in downstream jobs
-    - Trigger:
-        - This job is used in other pipelines but can be launched manually
-
-- Build-JDK8-linux_390-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Build-JDK8-linux_390-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Build-JDK8-linux_390-64_cmprssptrs)
-    - Description:
-        - Compiles java8 on linux_390-64_cmprssptrs
-        - Archives the SDK and test material for use in downstream jobs
-    - Trigger:
-        - This job is used in other pipelines but can be launched manually
-
-- Build-JDK9-linux_390-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Build-JDK9-linux_390-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Build-JDK9-linux_390-64_cmprssptrs)
-    - Description:
-        - Compiles java9 on linux_390-64_cmprssptrs
-        - Archives the SDK and test material for use in downstream jobs
-    - Trigger:
-        - This job is used in other pipelines but can be launched manually
+        - `.*(\n)?\bjenkins\s+compile\b\s*(((all|(([a-z]+(32)?,)*<platform>(,[a-z]+(32)?)*))\s*jdk<version>)(\s+depends.*)?))(\n)?.*`
+    - Test
+        - `.*(\n)?\bjenkins\s+test\s+<level>\b\s*(((all|([a-z]+(32)?,)*<platform>(,[a-z]+(32?))*)\s*jdk<version>)(\s+depends.*)?)(\n)?.*`
 
-- Build-JDK10-linux_390-64_cmprssptrs
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Build-JDK10-linux_390-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Build-JDK10-linux_390-64_cmprssptrs)
-    - Description:
-        - Compiles java10 on linux_390-64_cmprssptrs
-        - Archives the SDK and test material for use in downstream jobs
-    - Trigger:
-        - This job is used in other pipelines but can be launched manually
-
-- Build-JDK8-linux_ppc-64_cmprssptrs_le
-    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Build-JDK8-linux_ppc-64_cmprssptrs_le)](https://ci.eclipse.org/openj9/job/Build-JDK8-linux_ppc-64_cmprssptrs_le)
-    - Description
-        - Compiles java8 on linux_ppc-64_cmprssptrs_le
-        - Archives the SDK and test material for use in downstream jobs
-    - Trigger:
-        - This job is used in other pipelines but can be launched manually
+Where
+- `<version>` is the number of the supported release, e.g. 8 | 11 | n 
+- `<platform>` is xlinux | xlinuxlargeheap | zlinux | plinux | aix | win | win32
+- `<level>` is sanity | extended
```

Comment:
Having these example here is unnecessary imo because we've already listed all the values above.